frontend/src/components/header/reminders/reminders.module.scss

```diff
@@ -0,0 +1,36 @@
+.reminder {
+  display: flex;
+  align-items: center;
+  list-style: none;
+
+  li {
+    position: relative;
+    display: flex;
+    align-items: center;
+    margin-right: 21px;
+    cursor: pointer;
+
+    img {
+      width: 24px;
+      height: 24px;
+    }
+  }
+
+  &Counter {
+    position: absolute;
+    top: -5px;
+    right: -5px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 15px;
+    height: 15px;
+    background-color: #3765ff;
+    border-radius: 50%;
+
+    span {
+      color: #ffffff;
+      font-size: 10px;
+    }
```

Comment:
You could use the MUI's [`Badge`](https://mui.com/material-ui/react-badge/) component. But that's also good that you now know how to create badges in CSS :ok_hand: